pkg/controller/plan/validation.go

```diff
@@ -710,3 +736,27 @@ func (r *Reconciler) setupSecret(plan *api.Plan) (err error) {
 	plan.Referenced.Secret = &secret
 	return
 }
+
+func (r *Reconciler) checkOCPVersion(clientset kubernetes.Interface) error {
+	discoveryClient := clientset.Discovery()
+	version, err := discoveryClient.ServerVersion()
+	if err != nil {
+		return liberr.Wrap(err)
+	}
+
+	major, err := strconv.Atoi(version.Major)
+	if err != nil {
+		return liberr.Wrap(err)
+	}
+
+	minor, err := strconv.Atoi(version.Minor)
+	if err != nil {
+		return liberr.Wrap(err)
+	}
+
+	if major < 1 || (major == 1 && minor < 26) {
+		return liberr.New("OpenShift version is not supported")
```

Comment:
it would display OpenShift also for k8s then which is not that great - how about "Version of source provider" instead?